src/Sparrow/Json/JsonOperationContext.cs

```diff
@@ -35,9 +32,10 @@ public class JsonOperationContext : IDisposable
 
         private int _lastStreamSize = 4096;
 
-        public JsonOperationContext(UnmanagedBuffersPool pool)
+        public JsonOperationContext()
         {
-            Pool = pool;
+            _arenaAllocator = new ArenaMemoryAllocator();
+            _arenaAllocatorForLongLivedValues = new ArenaMemoryAllocator(16 * 1024);
```

Comment:
Why? We are going to be pooling the contexts, and the long lived is only used for property names.
We do have the case of the dynamic property names to consider, but that is something different.